src/Microsoft.Azure.WebJobs.Host/Loggers/FunctionInstanceLogger.cs

```diff
@@ -22,7 +22,7 @@ public FunctionInstanceLogger(ILoggerFactory loggerFactory)
 
         public Task<string> LogFunctionStartedAsync(FunctionStartedMessage message, CancellationToken cancellationToken)
         {
-            string traceMessage = string.Format(CultureInfo.InvariantCulture, "Executing '{0}' (Reason='{1}', Id={2})", message.Function.ShortName, message.FormatReason(), message.FunctionInstanceId);
+            string traceMessage = string.Format(CultureInfo.InvariantCulture, "Executing '{0}' (Reason='{1}', Id={2}, MessageId={3})", message.Function.ShortName, message.FormatReason(), message.FunctionInstanceId, message.MessageId);
```

Comment:
Based on my suggestions above, if message.TriggerMessage is not null/empty, we'd log a new log line after this, like $"Trigger Details: {message.TriggerDetails}"